index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My portfolio 👨‍💻 </title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+</head>
+<body>
+    
+    <header>
+        <h2>My logo</h2>
+       <nav>
+           <button type="button" class ="btn">&#9776;</button>
+       </nav>
+    </header>
+
+    <section class="headline-section">
+
+                <h1>I’m Abdifatah Yassen<br>
+                    Glad to see you!</h1>
+                    <p>I’m a software developer! I can help you<br> build a product , feature or website Look<br>
+                         through some of my work and <br>experience! If you like what you see and<br> have a project you need
+                        coded, don’t <br>hestiate to contact me.</p>
+                        <p id="text">LET’S CONNECT</p>
+          
+                <div class="social-icons">
+                    
+                    <i class="fab fa-twitter"></i>
+                    <i class="fab fa-facebook"></i>
+                     <i class="fab fa-twitter"></i>
+                        <i class="fab fa-instagram"></i>
+                         <i class="fab fa-youtube"></i>
+                         
+                    
+                </div>
```

Comment:
- [ ] Kindly make use of the list element in listing items on your page. For this project, you're required to use the `ul` and `li` tags for listing your social media links. And also, please make each icon be in an `a` tag so that you can pass that social media link on it.

Reply:
Done